pallets/nodle-staking/Cargo.toml

```diff
@@ -0,0 +1,60 @@
+[package]
+name = "pallet-nodle-staking"
+version = "2.0.1"
+authors = ['Eliott Teissonniere <git.eliott@teissonniere.org>, R.RajeshKumar <rajesh@nodle.co>']
+edition = "2018"
+description = "nodle's custome staking pallet, use a non inflationary reward system"
+readme = "README.md"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+frame-support = { version = "3.0.0", default-features = false }
+frame-system = { version = "3.0.0", default-features = false }
+pallet-balances = { version = "3.0.0", default-features = false }
+parity-scale-codec = { version = "2.0.0", default-features = false, features = ["derive"] }
+serde = { version = "1.0.101", optional = true }
+sp-std = { version = "3.0.0", default-features = false }
+sp-runtime = { version = "3.0.0", default-features = false }
+sp-staking = { version = "3.0.0", default-features = false }
+pallet-session = { version = "3.0.0", default-features = false, features = ["historical"] }
+pallet-authorship = { version = "3.0.0", default-features = false }
+log = { version = "0.4.14", default-features = false }
+
+# Optional imports for benchmarking
+frame-benchmarking = { version = "3.0.0", default-features = false, optional = true }
+pallet-poa = { default-features = false, path = "../poa", optional = true }
+
+[dev-dependencies]
+sp-core = { version = "3.0.0", default-features = false }
+sp-io = { version = "3.0.0", default-features = false }
+sp-tracing = { version = "3.0.0", default-features = false }
+pallet-timestamp = { version = "3.0.0", default-features = false }
+substrate-test-utils = { version = "3.0.0", default-features = false }
+
+[features]
+default = ["std"]
+std = [
+	"frame-support/std",
+	"frame-system/std",
+	"pallet-balances/std",
+	"parity-scale-codec/std",
+	"serde",
+	"sp-std/std",
+	"sp-runtime/std",
+	"sp-staking/std",
+	"pallet-authorship/std",
+	"pallet-session/std",
+	"pallet-poa/std",
+	"frame-benchmarking/std",
+	"log/std",
+]
+runtime-benchmarks = [
+	"frame-benchmarking",
+	"frame-system/runtime-benchmarks",
+	"frame-support/runtime-benchmarks",
+]
+test-migration = [
+	"pallet-poa",
+]
```

Comment:
Thought it would be context specific, gets used only when testing the runtime migration code. I am rethinking now.
Shall we make it part of dev-dependencies, without any feature flag ?

Reply:
Valid one Eliott, moved `pallet-poa` part of  `dev-dependencies`